sdk/ml/azure-ai-ml/azure/ai/ml/constants/_job/automl.py

```diff
@@ -54,65 +54,6 @@ class AutoMLTransformerParameterKeys(Enum):
     TF_IDF = "TfIdf"
     HASH_ONE_HOT_ENCODER = "HashOneHotEncoder"
 
-
-class ImageClassificationModelNames(Enum):
```

Comment:
if these are GA'ed constants, they can't be removed as it would be a breaking change.